src/pages/newblog/NewBlog.jsx

```diff
@@ -0,0 +1,104 @@
+    */
+
+
+function NewBlog() {
+    const {register, handleSubmit, setValue, formState: {errors}} = useForm();
+    const [apiData, setApiData] = useState("0");
+    const [error, setError] = useState("");
+    const [loading, toggleLoading] = useState(false);
+
+
+    useEffect(() => {
+        register("created");
+        register("readTime");
+        register("comments");
+        register("shares");
+    }, [register]);
+
+
+    const onSubmit = (data) => {
+        const standardizedData = {
+        ...data,
+        created: timeStamp(),
+        readTime: readTimer(data.content),
+        comments: 0,
+        shares: 0,
+        }
+        pushNumberOne(standardizedData, setError, setApiData, toggleLoading);
+    };
+
+
+    return (
+        <div className="page-container">
+            {loading && <p>Post versturen...</p>}
+            {apiData?.id > 0 &&
+                <div className="page-container">
+                    <h2>De blogpost is succesvol toegevoegd.</h2>
+                    <Link className="post-link" to={`/post/${apiData.id}`}>Je kunt deze hier bekijken<ArrowCircleRightIcon size={26}/></Link>
+                </div>}
+            {apiData.length > 0 &&
+                <div className="page-container">
+                    <h2>Begin hier met het maken van jouw nieuwe blog-post!</h2>
+                    {error && <h2 className="error">{error}</h2>}
+                    <form onSubmit={handleSubmit(onSubmit)}>
+                        <div className="form-control">
+                            <label>Titel</label>
+                            <input
+                                type="text"
+                                name="title"
```

Comment:
`name` is niet nodig bij react hook form, die geef je al mee aan de `...register("hier-komt-de-naam", {})`